plenum/server/replica.py

```diff
@@ -189,6 +189,7 @@ def cleanup(self):
 
         self._ordering_service.cleanup()
         self._checkpointer.cleanup()
+        self._view_change_service.cleanup()
```

Comment:
It would be great to have a unit test checking that we unsubscribe from all the services once Replica is removed